pkg/oc/cli/describe/chaindescriber_test.go

```diff
@@ -1,19 +1,17 @@
 package describe
 
 import (
-	"strings"
 	"testing"
 
 	"github.com/gonum/graph"
 	"github.com/gonum/graph/concrete"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/util/sets"
-	kapi "k8s.io/kubernetes/pkg/api"
-
-	"github.com/openshift/origin/pkg/client/testclient"
-	imagegraph "github.com/openshift/origin/pkg/image/graph/nodes"
 )
 
+/*
```

Comment:
@fabianofranz are you willing to fix these? @deads2k suggest to create a list filtered by the scheme in the fake client. The describer and project status now takes 5 different clients for the resources it produces the status for.
The test is reading a fixture file with mixed resources and we need 5 different fake clients fed by appropriate objects to make this test pass.